### ISSUE TITLE
TASK 2

### ISSUE THREAD

**Issue by MScientistCM** [MScientistCM/March2019ToV6Conversion#2]
d. Rename Holland --> Netherlands in euro.cfg. [Notepad]
e. Rename Holland --> Netherlands (four times) in nat_club.dat. [XVI32]
f. Rename Holland --> Netherlands (twice) in nation.dat. [XVI32]

**Comment by MScientistCM** [comment]
Task done and pushed.

**Comment by MScientistCM** [comment]
Oh no, I just realized the XVI32 tool 'find and replace' doesn't preserve the ammount of zeroed bytes after replacing an string. This means the zeroes need to be added/removed manually after replacing an string... So im reopening this ticket to add/remove the zeroes.

**Comment by MScientistCM** [comment]
done (correctly now, preserving number of bytes of the fields) and pushed.

**Comment by Dermotron** [comment]
I am wondering if this is the cause of the issues with the international managers? When added back from the Master to the active version, all the managers are the at the wrong Nation -1 e.g. Christian Panucci is the manager of Albania in real life but in the DB he is manager of Afghanistan. Even if I put the staff.dat from the March 2019 Data Update (unpatched) it's the same scenario

What I mean is, does the international manager of the Netherlands needs to be edited at this stage to prevent the " - 1 " scenario

**Comment by MScientistCM** [comment]
I will double check, but Im pretty sure the issue isn't occurring anymore since the last database you sent me yesterday night. I still didnt update the link in the beta thread though so you may be testing with the previous database?

**Comment by Dermotron** [comment]
Sorry, I posted here before seeing your post on the forum

**Comment by MScientistCM** [comment]
reuploading from sendspace the database version with all issues fixed: 
[Data.zip](https://github.com/MScientistCM/March2019ToV6Conversion/files/3014063/Data.zip)
I'll update the links in the beta thread.
Thanks @Dermotron

**Comment by MScientistCM** [comment]
Link for download the final beta version of the converted database, which should be the base for saturn create the v7 patch:

[DataFinalBetaV6March2019.zip](https://github.com/MScientistCM/March2019ToV6Conversion/files/3024857/DataFinalBetaV6March2019.zip)

The changes since the previous beta version are:

- Fix by Andrea for the itally positions.
- Fixed two MLS team lines in player_setup.cfg that got overwritten by some other changes.
- Added default eidos.rgn file as saturn did for v6 data.
- Added game.mbr and match.mbr file as saturn did for v6 data.
- Added the change from 11 to 12 in jap league rep in club_comp.dat which I thought I had added but I actually forgot :)

I reviewed all conversion steps from 1 to 9 and it contains all of them correctly. I also tested and is stable and working as expected. This database doesn't need any modifications and can be released as is for v7, unless saturn wants to change something in the database, or if some of the patches he will do will require to change the database.
